rebasehelper/application.py

```diff
@@ -224,7 +224,8 @@ def _prepare_spec_objects(self):
                 spec_file.download_remote_sources()
 
         # parse spec again with sources downloaded to properly expand %prep section
-        spec_file._update_data()
+        for spec_file in [self.spec_file, self.rebase_spec_file]:
+            spec_file._update_data()
```

Comment:
Can't this be actually moved into the loop just above?

Reply:
Yes, that would be probably better.